### DISCUSSION TITLE
How can we be of help?

### DISCUSSION THREAD

**Discussion by weeping-somnambulist** [go-task/task#507] [General]
As a newer Go dev, how can I help the Task team? What things would you like to see a volunteer work on first? Is there a roadmap or list of priorities that the core dev team would like to see new contributors work on? 

I use this project A LOT, and I'd like to see it continue to grow and mature. 

Let me know when you get a chance.

Thanks!
Sean

**Comment by andreynering** [comment]
Hi Sean!

I'd say we don't have a clear roadmap at the moment. Since v3.0.0 I've been focusing on bug fixes and small features as I haven't been having lots of free time available. Bigger features may eventually happen in the future, though.

I was going to say we have [first good issue](https://github.com/go-task/task/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) and [help wanted](https://github.com/go-task/task/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) labels, but by looking through it most of the clear issues already have opened PRs.

That said, you could look at the issue list to check if there's something you're willing to work on. I'd advice on asking first as not all features requests are going to be made, some are just ideas / discussion. Patches for bugs and very small features are more likely to be accepted than big changes.

If you feel something is missing you can open new issues for discussion as well.
